### ISSUE TITLE
Need to upgrade postgresql for solving vulnerability issues

### ISSUE THREAD

**Issue by mattcui** [cloudfoundry/bosh#1155]
There identified 2 vulnerability issues we need to solve:

https://exchange.xforce.ibmcloud.com/vulnerabilities/110627
https://exchange.xforce.ibmcloud.com/vulnerabilities/110626

According to the reports, we may need to upgrade postgresql:
```
Remedy:
Upgrade to the latest version of PostgreSQL (9.1.20, 9.2.15, 9.3.11, 9.4.6 and 9.5.1 or later), available from the PostgreSQL Web site. See References.
```

**Comment by cppforlife** [comment]
Lower priority since postgres is configured to listen on a local port: https://www.pivotaltracker.com/story/show/114884951

**Comment by dpb587-pivotal** [comment]
Upgraded and published in v255.4, 3213.
